package.json

```diff
@@ -44,18 +42,18 @@
     "is-url": "^1.2.4",
     "make-promises-safe": "^5.1.0",
     "mem": "^8.1.1",
-    "patch-package": "^6.4.7",
     "quick-lru": "^5.1.1",
-    "tiny-typed-emitter": "^2.1.0"
+    "tiny-typed-emitter": "^2.1.0",
+    "uuid": "^8.3.2"
```

Comment:
Maybe we can just use the existing function `generateId()` in `src/lib/utils.js` to avoid adding an extra dep? It should do exactly the same thing (just different string format, which should not affect the code).

Reply:
the reason for using uuid was to maintain interop with how prisma implements migrations. thought it would make sense to stick with it in case there was a case where interop should be maintained